pkg/build/types/parser/deffile.go

```diff
@@ -244,20 +280,20 @@ func populateDefinition(sections map[string]string, d *types.Definition) (err er
 
 	d.ImageData = types.ImageData{
 		ImageScripts: types.ImageScripts{
-			Help:        sections["help"],
-			Environment: sections["environment"],
-			Runscript:   sections["runscript"],
-			Test:        sections["test"],
-			Startscript: sections["startscript"],
+			Help:        *sections["help"],
```

Comment:
Why pass pointer here ?

Reply:
Sections is a map of pointer structs, so this is referencing a pointer to a struct so that i can be copied into the final definition struct. I used a pointer in the map so that I could easily modify the struct.